guess_the_number.py

```diff
@@ -1,6 +1,6 @@
 import random
 
-print('Guess the Number')
+print('Guess the Number game')
```

Comment:
Can you make the G in game uppercase?

Reply:
Sure I can work on it

Reply:
(Pretend we are tagging Clara) Completed the step for you!

Reply:
Looks great! thank you!